src/util/file_util.cpp

```diff
@@ -193,3 +193,12 @@ bool is_directory(const std::string &path)
 
 #endif
 }
+
+void create_directory(const std::string &path)
+{
+#ifdef _WIN32
+  _mkdir(path.c_str());
+#else
+  mkdir(path.c_str(), 0777);
```

Comment:
Is that really the umask we should be using? Also, in either case we should at bare minimum return the value that `mkdir` returned. The current approach necessarily hides errors.

Reply:
Now with comment about the umask, and with return value that matches what `std::filesystem::create_directory` does.